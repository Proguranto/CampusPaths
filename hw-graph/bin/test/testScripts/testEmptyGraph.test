# Tests graph creation/viewing with an empty graph

#Create a graph
CreateGraph graph1

#Print the nodes in the graph
ListNodes graph1