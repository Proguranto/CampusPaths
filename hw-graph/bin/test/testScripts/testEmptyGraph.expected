# Tests graph creation/viewing with an empty graph

#Create a graph
created graph graph1

#Print the nodes in the graph
graph1 contains: